daemon/daemon.go

```diff
@@ -113,6 +113,7 @@ const (
 // main config parameters by exiting and allowing itself to be restarted by the init
 // daemon.
 func Run(configFile string) {
+	log.Infof("OSENV: %v", os.Environ())
```

Comment:
Concerned that we might be leaking sensitive information into the log here.  Do we really need these logs?

Reply:
ug sorry, that was left over, I'll remove these